core/crypto/bls_provider/bls_provider_impl.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CRYPTO_BLS_PROVIDER_IMPL_HPP
+#define CRYPTO_BLS_PROVIDER_IMPL_HPP
+
+#include "crypto/bls_provider/bls_provider.hpp"
+
+namespace fc::crypto::bls {
+  class BlsProviderImpl : public BlsProvider {
+   public:
+    outcome::result<KeyPair> GenerateKeyPair() const override;
```

Comment:
Here and in other place please always use camelCase from small letter for methods names